libraries/chain/asset_evaluator.cpp

```diff
@@ -375,12 +370,6 @@ void_result asset_update_issuer_evaluator::do_evaluate(const asset_update_issuer
               "Incorrect issuer for asset! (${o.issuer} != ${a.issuer})",
```

Comment:
Please add a comment on line 293 why the HARDFORK_572 check has to stay.

Reply:
Still investigating as part of #1743